---a/net/minecraft/network/status/INetHandlerStatusClient.java
+++b/net/minecraft/network/status/INetHandlerStatusClient.java
@@ -4,10 +4,10 @@
 import net.minecraft.network.status.server.S00PacketServerInfo;
 import net.minecraft.network.status.server.S01PacketPong;
 
-public interface INetHandlerStatusClient extends INetHandler
-{
+public interface INetHandlerStatusClient extends INetHandler {
+
     void handleServerInfo(S00PacketServerInfo packetIn);
 
     void handlePong(S01PacketPong packetIn);
-}
+}
